lib/nexthop_group.c

```diff
@@ -706,6 +706,30 @@ DEFPY(no_nexthop_group_resilience,
 	return CMD_SUCCESS;
 }
 
+DEFPY(nexthop_group_recursive,
+      nexthop_group_recursive_cmd,
+      "[no] recursive",
+      NO_STR
+      "Allow recursive resolution\n")
+{
+	VTY_DECLVAR_CONTEXT(nexthop_group_cmd, nhgc);
+	uint32_t flags;
+
+	flags = nhgc->flags;
+
+	if (no)
```

Comment:
is it possible to add a topotest like was proposed in https://github.com/FRRouting/frr/pull/16028/commits/e60cd344d5c26c26b0e6bcaac169a4420bbd21f4 ?